scripts/scripts.js

```diff
@@ -138,6 +142,8 @@ function loadDelayed() {
   // eslint-disable-next-line import/no-cycle
   window.setTimeout(() => import('./delayed.js'), 3000);
   // load anything that can be postponed to the latest here
+
+  loadCSS(`${window.hlx.codeBasePath}/styles/styles.css`);
```

Comment:
Was this to ensure the base styles load in before the element-styling styles?

Reply:
Sort of. I removed the explicit link in the html files because the regular styles were loading in before the lazy styles. This also ensures the base styles load before component level styles.